src/MSALWrapper/AuthFlow/Broker.cs

```diff
@@ -73,65 +70,46 @@ private enum GetAncestorFlags
         protected override string Name() => NameValue;
 
         /// <inheritdoc/>
-        protected override async Task<(TokenResult, IList<Exception>)> GetTokenInnerAsync()
+        protected override async Task<TokenResult> GetTokenInnerAsync()
         {
             IAccount account = await this.pcaWrapper.TryToGetCachedAccountAsync(this.preferredDomain)
                  ?? PublicClientApplication.OperatingSystemAccount;
 
-            TokenResult tokenResult = null;
-            try
-            {
-                tokenResult = await CachedAuth.GetTokenAsync(
-                    this.logger,
-                    this.scopes,
-                    account,
-                    this.pcaWrapper,
-                    this.errors);
-
-                if (tokenResult != null)
-                {
-                    return (tokenResult, this.errors);
-                }
+            TokenResult tokenResult = await CachedAuth.GetTokenAsync(
+                this.logger,
+                this.scopes,
+                account,
+                this.pcaWrapper,
+                this.errors);
 
-                try
-                {
-                    tokenResult = await TaskExecutor.CompleteWithin(
-                        this.logger,
-                        this.interactiveAuthTimeout,
-                        $"{this.Name()} interactive auth",
-                        this.GetTokenInteractive(account),
-                        this.errors).ConfigureAwait(false);
-                }
-                catch (MsalUiRequiredException ex)
-                {
-                    this.errors.Add(ex);
-                    this.logger.LogDebug($"Initial {this.Name()} auth failed. Trying again with claims from exception.\n{ex.Message}");
-
-                    tokenResult = await TaskExecutor.CompleteWithin(
-                        this.logger,
-                        this.interactiveAuthTimeout,
-                        $"{this.Name()} interactive auth (with extra claims)",
-                        this.GetTokenInteractiveWithClaims(ex.Claims),
-                        this.errors).ConfigureAwait(false);
-                }
-            }
-            catch (MsalServiceException ex)
+            if (tokenResult != null)
             {
-                this.logger.LogWarning($"MSAL Service Exception! (Not expected)\n{ex.Message}");

```

Comment:
Is it okay if we miss out on providing the warnings like this?

Reply:
@mvanchaa - In this case, we have a token so we don't need any warning. We can return early directly. The logic for specific exceptions have been moved below.

Reply:
We still warn, in the base class's method. So all warnings done consistently.

Reply:
I only see the debug statements though :o @kyle-rader-msft